datumaro/plugins/data_formats/video.py

```diff
@@ -124,3 +127,101 @@ def __iter__(self):
     def get(self, id, subset=None):
         assert subset == self._subset, "%s != %s" % (subset, self._subset)
         return super().get(id, subset or self._subset)
+
+
+class VideoKeyframesImporter(VideoFramesImporter):
+    """
+    Reads video frames as a dataset.
+    """
+
+    @classmethod
+    def build_cmdline_parser(cls, **kwargs):
+        parser = super().build_cmdline_parser(**kwargs)
+        parser.add_argument(
+            "-t",
+            "--threshold",
+            type=float,
+            default=0.3,
+            help="Similarity threshold (default: %(default)s)",
+        )
+
+        parser.add_argument(
+            "-resize",
```

Comment:
```suggestion
            "-r",
```

It's common practice to combine a single hyphen with a single character. Please refer to https://serverfault.com/a/387936/776078.

Reply:
thank you, I corrected